crates/l2/contracts/src/l1/OnChainProposer.sol

```diff
@@ -313,6 +313,11 @@ contract OnChainProposer is
             );
         }
 
+        require(
+                ICommonBridge(BRIDGE).withinProcessingDeadline(),
+                "OnChainProposer: exceeded privileged transaction inclusion deadline"
+            );
```

Comment:
I'm concerned this could result in the sequencer being required to post a batch that cannot be posted due to size constraints. What are we doing to avoid this?